force-app/main/default/sharingRules/Contact.sharingRules-meta.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SharingRules xmlns="http://soap.sforce.com/2006/04/metadata">
+    <sharingOwnerRules>
+        <fullName>ANZ_Account_Visibility</fullName>
+        <accessLevel>Edit</accessLevel>
+        <description>Everyone under a given market leader role can see/edit records created by anyone in that branch of the hierarchy</description>
+        <label>ANZ Account Visibility</label>
+        <sharedTo>
+            <roleAndSubordinatesInternal>ANZ_Market_Leader</roleAndSubordinatesInternal>
+        </sharedTo>
+        <sharedFrom>
+            <roleAndSubordinatesInternal>ANZ_Market_Leader</roleAndSubordinatesInternal>
+        </sharedFrom>
+    </sharingOwnerRules>
+    <sharingOwnerRules>
+        <fullName>CAN_Account_Visibility</fullName>
+        <accessLevel>Edit</accessLevel>
+        <description>Everyone under a given market leader role can see/edit records created by anyone in that branch of the hierarchy</description>
+        <label>CAN Account Visibility</label>
+        <sharedTo>
+            <roleAndSubordinatesInternal>CAN_Market_Leader</roleAndSubordinatesInternal>
+        </sharedTo>
+        <sharedFrom>
+            <roleAndSubordinatesInternal>CAN_Market_Leader</roleAndSubordinatesInternal>
+        </sharedFrom>
+    </sharingOwnerRules>
+    <sharingOwnerRules>
+        <fullName>EUR_Account_Visibility</fullName>
+        <accessLevel>Edit</accessLevel>
+        <description>Everyone under a given market leader role can see/edit records created by anyone in that branch of the hierarchy</description>
+        <label>EUR Account Visibility</label>
+        <sharedTo>
+            <roleAndSubordinatesInternal>EUR_Market_Leader</roleAndSubordinatesInternal>
+        </sharedTo>
+        <sharedFrom>
+            <roleAndSubordinatesInternal>EUR_Market_Leader</roleAndSubordinatesInternal>
+        </sharedFrom>
+    </sharingOwnerRules>
+    <sharingOwnerRules>
+        <fullName>Global_Team</fullName>
+        <accessLevel>Edit</accessLevel>
+        <description>Everyone under a given market leader role can see/edit records created by anyone in that branch of the hierarchy</description>
+        <label>Global Team</label>
+        <sharedTo>
+            <roleAndSubordinatesInternal>Global_Team</roleAndSubordinatesInternal>
+        </sharedTo>
+        <sharedFrom>
+            <roleAndSubordinatesInternal>Global_Team</roleAndSubordinatesInternal>
+        </sharedFrom>
```

Comment:
@rthomp-prosci -- This would effectively grant edit access to all contacts to all users within the hierarchy. Let's remove this rule.